src/openrct2/world/TileElement.h

```diff
@@ -65,6 +65,8 @@ struct TileElementBase
     uint8_t base_height;      // 2
     uint8_t clearance_height; // 3
 
+    uint8_t testPad[8];
```

Comment:
This struct shouldn't have any padding. Padding should be added to all the other structs.

Reply:
It was just a test, but I will change it.